ge/org.osate.ge.gef.ui/src/org/osate/ge/gef/ui/editor/AgeEditor.java

```diff
@@ -465,10 +466,21 @@ public void afterModelChangeNotification() {
 	private final DiagramModificationListener diagramModificationListener = new DiagramModificationAdapter() {
 		@Override
 		public void modificationsCompleted(final ModificationsCompletedEvent e) {
+			final List<?> diagramNodes = selectionProvider.getSelection().toList();
+			// Check if selected elements are visible
+			if (diagramNodes.stream()
+					.map(element -> element instanceof DiagramNode ? gefDiagram.getSceneNode((DiagramNode) element)
+							: null)
+					.anyMatch(Predicates.isNull())) {
```

Comment:
Don't use Predicates.isNull(). There is a standard function that will serve the same purpose. Objects::isNull.